### PR TITLE
Adaptivity partners

### PR DESCRIPTION
**Pull request by MaksBrat**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/features/StreetcodePage/PartnersBlock/Partners.styles.scss [hunk 1]
```diff
@@ -19,4 +20,20 @@
             @include mut.flex-centered();
         }
     }
+}
+
+@media screen and (max-width: 1320px){
+    .partnersWrapper{
+        padding: 0 !important;  
+    }
+}
```
Comment (Kotusyk) [inline src/features/StreetcodePage/PartnersBlock/Partners.styles.scss:29]: Just a question: Does it a crutch, or it's just my wrong suppose?

File: src/app/common/components/modals/Donates/DonatesModal.styles.scss [hunk 2]
```diff
@@ -124,4 +128,85 @@
...
+      }
+  
+      .donatesBtnContainer{
+        @include mut.sized(408px, 100%);
+        margin-bottom: f.pxToRem(16px);
+  
+        > * {
+          width: 120px !important;
+          height: 37px !important;
+
```
Comment (NadiaKishchuk) [inline src/app/common/components/modals/Donates/DonatesModal.styles.scss:181]: don`t forget about using pxToRem function

**Review by NadiaKishchuk — APPROVED** [review]
nice work

File: yarn.lock [hunk 3]
```diff
@@ -9304,14 +9310,22 @@
...
+  resolved "https://registry.npmjs.org/react-leaflet/-/react-leaflet-4.2.0.tgz"
+  integrity sha512-9d8T7hzYrQA5GLe3vn0qtRLJzQKgjr080NKa45yArGwuSl1nH/6aK9gp7DeYdktpdO1vKGSUTGW5AsUS064X0A==
+=======
 "react-leaflet@^3.0.0 || ^4.0.0", "react-leaflet@^4.0.0", "react-leaflet@^4.2.0":
   "integrity" "sha512-9d8T7hzYrQA5GLe3vn0qtRLJzQKgjr080NKa45yArGwuSl1nH/6aK9gp7DeYdktpdO1vKGSUTGW5AsUS064X0A=="
   "resolved" "https://registry.npmjs.org/react-leaflet/-/react-leaflet-4.2.0.tgz"
   "version" "4.2.0"
+>>>>>>> fixed
   dependencies:
     "@react-leaflet/core" "^2.1.0"
```
Comment (Kotusyk) [inline yarn.lock:9330]: It will be better if you remove all "HEAD, ===, fixed"

**Review by Kotusyk — APPROVED** [review]

**Review by Tysyatsky — COMMENTED** [review]
remove >>>> HEAD in yarn.lock and you can merge it in fixed.

**Pull request merged**